examples/cms-tina/components/cover-image.js

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames'
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function CoverImage({ title, src, slug, height, width }) {
+  const image = (
+    <Image
+      src={src}
+      alt={`Cover Image for ${title}`}
+      className={cn('shadow-sm', {
+        'hover:shadow-md transition-shadow duration-200': slug,
+      })}
+      layout="responsive"
+      width={width}
+      height={height}
+    />
+  )
+  return (
+    <div className="sm:mx-0">
+      {slug ? (
+        <Link as={`/posts/${slug}`} href="/posts/[slug]">
```

Comment:
We no longer need to use `as` here - can just use `href`

Reply:
Right back at ya. this is your code.